src/models/neural-clean-conversation.ts

```diff
@@ -0,0 +1,36 @@
+import Message from './message';
+import Attributes from './attributes';
+import File from './file';
+
+export default class NeuralCleanConversation extends Message {
+  conversation?: string;
+  modelVersion?: string;
+
+  extractImages(): Promise<File[]> {
+    const f: File[] = [];
+    if (this.conversation) {
+      const regex = /[(']cid:(.)*[)']/g;
```

Comment:
would be nice to move this on the top of the file

Reply:
Good catch!